common/views/components/AppContext/AppContext.tsx

```diff
@@ -28,9 +30,13 @@ export const AppContextProvider: FunctionComponent<AppContextProviderProps> = ({
 }: AppContextProviderProps): ReactElement<AppContextProviderProps> => {
   const [isEnhanced, setIsEnhanced] = useState(appContextDefaults.isEnhanced);
   const [isKeyboard, setIsKeyboard] = useState(appContextDefaults.isKeyboard);
+  const [isFullSupportBrowser, setIsFullSupportBrowser] = useState(
+    appContextDefaults.isFullSupportBrowser
+  );
 
   useEffect(() => {
     setIsEnhanced(true);
+    setIsFullSupportBrowser('IntersectionObserver' in window);
```

Comment:
Could you leave a note explaining why we use this and what browsers it won't support (or a link to https://caniuse.com/intersectionobserver)?